guide/private-key-management/multi-key.md

```diff
@@ -61,6 +67,7 @@ A software wallet application or coordination software initiates a multi-sig wal
 {% include fact/cons.html %}
 
 - Has significant complexity and op-sec burden for multiple private keys, each of which need a good backup scheme
+- Not compatible with Lightning nodes
```

Comment:
Should this be "wallet" instead of "node"?

Reply:
My reasoning for using node instead of wallet was for cases when the application uses lndhub or similar, like Bluewallet. In that case I don't think the key matters. But I may be wrong. I can live with calling it wallet.

Reply:
Not sure, TBH. Just stood out because we use the word wallet everywhere.

Reply:
I'll update it to 'Lightning node wallets' 😉